apache/cloudstack
# Improve default network offering setting for egress policy

### Description

This PR addresses #5957














### Types of changes


### Feature/Enhancement Scale or Bug Severity

#### Feature/Enhancement Scale




### Screenshots (if appropriate):


### How Has This Been Tested?
1. Create an Isolated Network offering setting the default egress policy (if firewall service is checked) - true / false
2. Create a network using that offering
3. Notice in the details tab of the created network you will find details of the policy:
![image](https://user-images.githubusercontent.com/10495417/154954642-53d67a68-333f-42cc-9827-9e2a957e2162.png)
4. Go to the egress rules tab, there's a note for user to understand the implication of the policy / rules they add:
![image](https://user-images.githubusercontent.com/10495417/154954974-7aee5024-9081-43e4-a298-2b18a51d5ef2.png)
